tests/test_vcs_git/test_vcs_git.py

```diff
@@ -0,0 +1,42 @@
+from os.path import dirname, join
+from shutil import copyfile
+
+from dxr.testing import DxrInstanceTestCase
+
+from nose import SkipTest
+from nose.tools import ok_
+
+
+class GitTests(DxrInstanceTestCase):
+    """Tests that the diff links for files go somewhere helpful"""
```

Comment:
Those "in" and "not in" assertions don't give you much to go on. You could (temporarily?) write a better assert_in() helper that dumps the entirety of what you're comparing against.